src/Data/DateTime.purs

```diff
@@ -0,0 +1,216 @@
+module Ocelot.Data.DateTime where
+
+import Prelude
+
+import Data.Array (drop, find, reverse, sort, take)
+import Data.Date (Date, Day, Year, canonicalDate, lastDayOfMonth, year, month, day, Weekday(..), weekday)
+import Data.DateTime (DateTime(..), Hour, Millisecond, Minute, Month(..), Second, Time(..))
+import Data.DateTime (adjust, date) as DT
+import Data.DateTime.Instant (fromDate, toDateTime)
+import Data.Either (either)
+import Data.Enum (class Enum, fromEnum, pred, succ, toEnum)
+import Data.Formatter.DateTime (Formatter)
+import Data.Formatter.DateTime as Formatter
+import Data.FunctorWithIndex (mapWithIndex)
+import Data.List (List(..), (:))
+import Data.Maybe (Maybe(..), fromJust, maybe)
+import Data.String.Regex (parseFlags, regex, replace)
+import Data.Time (Hour, Millisecond, Minute, Second, Time(..))
+import Data.Time (adjust) as T
+import Data.Time.Duration (Days(..), Hours(..), Minutes(..), negateDuration)
+import Data.Tuple (Tuple(..), fst, snd)
+import Partial.Unsafe (unsafePartial)
+
+
+----------
+-- Defaults
```

Comment:
Why do we need these defaults? Seems like we'd be better off with a `Maybe` than setting to such odd dates and times in the case we don't have an actual date at hand.

Reply:
I prefer a default to carrying around a maybe everywhere, but that aside, some functions we use  such as the formatters, require a `DateTime` so if we only have `Date` or `Time`, we can use one of these defaults to construct a full `DateTime`